### DISCUSSION TITLE
Tarpits as hive sensor deployment

### DISCUSSION THREAD

**Discussion by kauedg** [telekom-security/tpotce#1352] [Q&A] [answered]
Is it possible to make the tarpit deployment type send logs to a hive server, as if it was a sensor itself?

**Comment by kauedg** [comment]
@t3chn0m4g3 as this was turned into a discussion, would you be interested in building a command line to break the 'monolitic' docker-compose into a selectable model? Something like 

`# tpot enable cowrie --service ssh`

Before finding your project I was working on a remote administration CLI but I'm not sure if it's the kind of path you want the project to take...

**Reply by t3chn0m4g3** [reply, marked as answer]
No, but you can isolate one for ddospot. In the `docker/ddospot` folder you will find everything you need.
